libs/ledger-live-common/src/families/bitcoin/wallet-btc/explorer/index.ts

```diff
@@ -1,9 +1,9 @@
-import type { AxiosRequestConfig } from "axios";
+import type { AxiosRequestConfig, AxiosResponse } from "axios";
 import axios, { AxiosInstance } from "axios";
 import axiosRetry, { isNetworkOrIdempotentRequestError } from "axios-retry";
 import BigNumber from "bignumber.js";
 import genericPool, { Pool } from "generic-pool";
-
+import https from "https";
```

Comment:
Unfortunately I think `https` is not available on React Native env. This is why there is that weird use of `require`... I think we need to keep it.

I remembered that super old issue back when we were working on bitcoin-js :smile: 
https://github.com/LedgerHQ/wallet-btc/pull/44/files

Reply:
good point. I've reverted this modification.